src/pages/components/versions.mdx

```diff
@@ -0,0 +1,18 @@
+---
+title: Versions
+description: >
+  Description will be here
+---
+
+Garden website release documentation
+
+- [Versions 7](https://garden-v7.netlify.app/)
+- [Versions 6](https://garden-v6.netlify.app/)
+
+import { graphql } from 'gatsby';
+
+export const pageQuery = graphql`
+  query($fileAbsolutePath: String) {
+    ...SidebarPageFragment
+  }
+`;
```

Comment:
Feels like this could benefit from being under a `/pages/components/foundations/` subdirectory.

Reply:
Visually it's [there in the sidebar](https://share.getcloudapp.com/d5uWYnQp), would we want to commit to that URL slug structure as well?

Reply:
I guess we're keeping the URL structure pretty flat in the other pillars. After looking closer, this should probably stay the way you have it.